### PR TITLE
Issue #321: paired lung ct review

### PR DESCRIPTION
**Pull request by NMontanaBrown**
# Description

Review:

- [x] Author - added @s-sd as contact
- [x] Disclaimer - link to be fixed with future merge
- [x] Application - description included
- [x] Data - open accessible, linked.
- [x] Instructions - I may update these as it instructs the user to run in a given order but doing so would do a training from scratch and then test on pre-downloaded model, which may be confusing.
    - [x] Download - working
    - [x] Train - seems to run, but cannot afford to run 500 epochs on CPU... workarounds, suggestions for test?
    - [x] Predict - working
- [x] Pre-trained Model - available
- [x] Tested version - included. Update to latest commit tested?
- [x] Contact - included.
- [x] References - included.

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by NMontanaBrown** [comment]
Problem with visualisation, overlapping titles:
visualisation

Modifying such that titles don't overlap

**Comment by NMontanaBrown** [comment]
Fixed to:

visualisation

**Comment by s-sd** [comment]
@NMontanaBrown, just saw this. I might have a suggestion to test train. I think we can run it for a small number of epochs like 4 with save_freq of 2 to see that a model is being saved. Then we can load in that saved model with predict to see that it can be loaded in correctly. I think that would show that train is working as intended for that demo.

File: demos/paired_ct_lung/README.md [hunk 1]
```diff
@@ -44,18 +44,18 @@ following order:
...
-  - Train a network using deepreg
+    avialable with DeepReg and the config file for the data is given in the demo folder)
+  - Train a network using DeepReg
 - Run the demo_predict.py script: This script does the following:
   - Use the pretrained network to make predictions for the test set
   - Use the predicitons to plot the results (the path to the images generated in the
-    logs will need to be sepcified, look at comments in the script to chnage this)
+    logs will need to be specified, look at comments in the script to chnage this)
 
 ## Data
```
Comment (mathpluscode) [inline demos/paired_ct_lung/README.md:60]: Can you move the data section and add contact as in https://raw.githubusercontent.com/DeepRegNet/DeepReg/master/docs/source/demo/readme_template.md

thx!

**Review by mathpluscode — CHANGES_REQUESTED** [review]
please move data section and add contact section, then remove WIP once ready ;) thx

**Comment by mathpluscode** [comment]
@NMontanaBrown  is this branch ready for merge? We'd better merge them now before switch default branch to main as PRs will have to be retargeted https://github.com/github/renaming

**Review by mathpluscode — APPROVED** [review]

**Review by YipengHu — APPROVED** [review]

**Pull request merged**